api/contrib/envoy/extensions/private_key_providers/cryptomb/v3alpha/cryptomb.proto

```diff
@@ -0,0 +1,44 @@
+syntax = "proto3";
+
+package envoy.extensions.private_key_providers.cryptomb.v3alpha;
+
+import "envoy/config/core/v3/base.proto";
+
+import "google/protobuf/duration.proto";
+
+import "udpa/annotations/sensitive.proto";
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.private_key_providers.cryptomb.v3alpha";
+option java_outer_classname = "CryptombProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: CryptoMb private key provider]
+// [#extension: envoy.tls.key_providers.cryptomb]
+
+// A CryptoMbPrivateKeyMethodConfig message specifies how the CryptoMb private
+// key provider is configured. The private key provider provides `SIMD`
+// processing of ECDSA sign operations and RSA sign and decrypt operations. It
```

Comment:
Update docs: this now does "normal" processing for ecdsa.

Reply:
Ok, fixed.